RoboJackets/robocup-software
# play/defensive_clear mypy fix

## Description
Fixed mypy typing errors in the file play/defensive_clear.

## Steps to test
Find "Pull request checks / mypy" and click details.

Expected result: There should be no mypy errors for play/defensive_clear
